### ISSUE TITLE
Do a simple project for the community-fusions in Flanders

### ISSUE THREAD

**Issue by joostschouppe** [osmbe/community_planning#2]
There are a lot of street- and address changes coming up in some Flemish communities, caused by the fusions of some of them. It's an easy challenge for us, so why not take it a step further:

- [x] make a [wiki page](https://wiki.openstreetmap.org/wiki/WikiProject_Belgium/Municipality_Fusions) with more info for mappers, invent a tag to be sure about which objects to retag: we're using proposed:name=* and "fixme:name"="WikiProject_Belgium/Municipality_Fusions"
- [x] collect all the info now, from the websites or by contacting the local gov
- [x] add them to OSM as proposed:name
- [x] make a map showing the streets that have a proposed:name. [Map can be found here]( https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies)
- [x] check with local governments if the results are correct
- [x] on Dec 31st, change name to old_name and proposed:name to name
- [x] create new admin_level=8 units. Decide what to do with the old ones. (details in the wiki)
- [x] change the showcase map to show old_name for the streets that have been renamed
- [x] document the whole process to show off how extremely easy it is to do this kind of stuff with OSM
- [ ] make sure people notice
- [ ] make a osm.be project page or at least a blog post (don't forget to get in touch with the mapcontrib people!)
- [x] do the same for addr:street and associatedStreets, but keep the maps separate
- [ ] success!

I think this would be a project that is really easy, and that could show how cool we are at conferences and the like. Pick any of the subtasks, so we can mark that last one :) 

Local overview

| Municipality (link to data) | Data status | Mapping status | Feedback status |
| ------------- |:-------------:|:-------------:| -----:|
| [Kruishoutem-Zingem](http://fusiekz.be/2018/03/26/nieuwe-straatnamen/) | All but one name available | mapped, [list](https://docs.google.com/spreadsheets/d/1XPQUPZwME1LZ8fsckreXfPQ_luyI0CBZVPvLHOBZjR0/edit?usp=sharing), waiting for feedback | |
| [Aalter-Knesselare](https://www.samensterkerbeter.be/nieuws/nieuwe-straatnamen) | names and map available | [mapped](https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/12/51.0876/3.4181) | invited for meeting with Aalter to check, Knesselare confirmed correct |
| [Overpelt-Neerpelt](https://www.neerpelt.be/buurtbabbels) | street names available as of now | [mapped](https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/13/51.208507338932264/5.424259192077444) |  Pelt answered with thanks and confirmation, Overpelt pointed out a few forgotten streets. All done  |
| [Oudsbergen](https://www.oudsbergen.be/veelgestelde-vragen#straatnamen) (Meeuwen-Gruitrode + Opglabeek) | list available  | mapped [list](https://docs.google.com/spreadsheets/d/16j5Dp0zCtUDbIo-VPvS5ayN3hjpRo3PUz4QCT7YFzRE/edit?usp=sharing), [map](https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/12/51.080600367635945/5.561416632262991) | reviewed and confirmed changes, still one streetname TBD |
| [Deinze-Nevele](http://www.fusiedeinzenevele.be/lijst) | All data mailed to Joost | [done](https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/12/51.01930542354697/3.541854866780341) | Mapped without further problems |
| [Puurs - Sint-Amands](https://puurs-sint-amands.be/nieuwe-straatnamen/) | got the data from local admin | mapped | waiting on feedback |
| [Lievegem](https://lievegem.be/volg-de-fusie/straatnamen-wijziging2) (Lovendegem-Waarschoot-Zomergem) | data OK | [mapped, one open issue](https://docs.google.com/spreadsheets/d/12vB5AnPuiGf_W3x7i8SmedWeusnhIzWfYwPozoZNEqk/edit?usp=sharing) | waiting for feedback |

**Comment by joostschouppe** [comment]
Collecting info:
- [ ] [Kruishoutem-Zingem](http://fusiekz.be/2018/03/26/nieuwe-straatnamen/). New names available Sept 2018; but we could already mark proposed:name:TBD
-  [x] [Aalter-Knesselare](https://www.samensterkerbeter.be/nieuws/nieuwe-straatnamen): names and map available
-  [x] [Overpelt-Neerpelt](https://www.neerpelt.be/buurtbabbels): street names available as of now
-  [x] [Oudsbergen](https://www.oudsbergen.be/veelgestelde-vragen#straatnamen) (Meeuwen-Gruitrode + Opglabeek): list available 
-  [x] [Deinze-Nevele](http://www.fusiedeinzenevele.be/lijst): affected list available, but no new names available. Some more special situations. Preliminary list received by e-mail.
-  [ ] [Puurs - Sint-Amands](https://puurs-sint-amands.be/nieuwe-straatnamen/): there is a proposal, it should have been confirmed already but site not updated
-  [ ] [Lievegem](https://lievegem.be/volg-de-fusie/straatnamen-wijziging2) (Lovendegem-Waarschoot-Zomergem): a proposal is almost complete, but not official yet (should be available already)

See also: [official Flemish data](https://overheid.vlaanderen.be/CRAB-Gemeentelijke-fusies#adreswijzigingen-pro-actief-verwerken)

**Comment by joostschouppe** [comment]
Mapping work:
- [ ] Nevele-Deinze: in progress

**Comment by pietervdvn** [comment]
Mapping work

- Overpelt-Neerpelt

Overview here: https://docs.google.com/spreadsheets/d/1pY0kvzqCAChrqzDcQDQHhuTG2yqEBud72V5gMBY4y-8/edit#gid=0

**Comment by pietervdvn** [comment]
Working on Aalter-Knesselare

**Comment by pietervdvn** [comment]
Problem: https://www.openstreetmap.org/way/514611477

According to [this map](https://www.google.com/maps/d/viewer?mid=1LT4v9HYVN1zI-4gN4t1AdrJoTLSvqwTM&ll=51.11350143335531%2C3.4298908710479736&z=17), it will be renamed, but not according to the official list. I did contact them.

**Comment by pietervdvn** [comment]
Problem: https://www.openstreetmap.org/way/117735128

Noted differently on their map. Prolly error in GMaps. Contacted them

Edit: OSM confirmed correct, prepared change is correct

**Comment by pietervdvn** [comment]
Because @joostschouppe didn't mention already:


There is now an interactive map of the work already done: https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies

It works by querying things with a "proposed:name" and "fixme:name"="WikiProject_Belgium/Municipality_Fusions". So only if the you add those tags, your edits will show up.

**Comment by pietervdvn** [comment]
I send the map to Aalter + Knesselare, with a request to check

> Hallo,
> 
> We zijn binnen OpenStreetMap jullie naamswijzigingen aan het voorbereiden. De wijzigingen die we op 1 januari gaan doorvoeren, kunnen jullie op deze kaart vinden:
> https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies/13/51.1042/3.4482#
> 
> Kunnen jullie eens nakijken ofdat deze wijzigingen correct zijn? Zoja, dan hebben jullie meteen up to date kaarten op de vele apps die OSM gebruiken.
> 
> Bij fouten, laat dit gerust aan mij weten, dan passen we het nog aan.
> 
> MVG, Pieter
> 
> PS: jullie hebben natuurlijk al een kaartje, maar voel jullie vrij om deze kaart verder te verspreiden.

**Comment by pietervdvn** [comment]
Mailed Pelt

**Comment by pietervdvn** [comment]
Mailed Pelt

**Comment by joostschouppe** [comment]
Contacted Oudsbergen:

> Beste,
> We zijn binnen OpenStreetMap de adreswijzigingen in het kader van de fusie aan het voorbereiden. Ons datamodel laat toe om de toekomstige wijzigingen nu al aan te brengen als "data die in toekomst geldig is". Op 1 januari gaan we dus live met de nieuwe toestand met een druk op de knop. Dit stroom dan door naar OpenStreetMap.org en de honderden apps en websites die onze data hergebruiken.
> 
> Op basis van de info op uw website, hebben we dit reeds gedaan, wat u kunt visuelasieren op deze kaart: 
https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/12/51.07762383554728/5.5798873817548165
Door op een gemarkeerde straat te klikken, kan u de oude en nieuwe naam zien. De ene straat die nog beslist moet worden, is aangeduid in het rood.
> 
> Graag horen we van u of we dit volledig juist hebben gedaan. Het staat u volledig vrij de data te hergebruiken (deze is eenvoudig downloadbaar), of deze kaart in te bouwen in uw website (dat kan via link of via de share-knop als iFrame).
> 
> Met vriendelijke groet,
> Joost Schouppe
> Board member, OpenStreetMap Belgium
> www.osm.be

**Comment by joostschouppe** [comment]
Lievegem finished, just one minor issue. Contacted them with this letter:

> Beste,
> We zijn binnen OpenStreetMap de adreswijzigingen in het kader van de fusie aan het voorbereiden. Ons datamodel laat toe om de toekomstige wijzigingen nu al aan te brengen als "data die in toekomst geldig is". Op 1 januari gaan we dus live met de nieuwe toestand met een druk op de knop. Dit stroomt dan door naar OpenStreetMap.org en de honderden apps en websites die onze data hergebruiken.
> 
> Op basis van de info op uw website, hebben we dit reeds gedaan. Dit kunt u visualiseren op deze kaart:
https://www.mapcontrib.xyz/t/2d975c-Geplande_adreswijzigingen_gemeentefusies#position/12/51.12169156110325/3.6162872356362645
> Door op een gemarkeerde straat te klikken, kan u de oude en nieuwe naam zien. We hebben hierbij slechts twee problemen vastgesteld. Op de website is sprake van Schauwbroeckstraat, maar in CRAB staat dit geschreven als 'broek', niet 'broeck'. Belangrijker: we begrepen niet wat u bedoelt met "Lievestraat (deel Grote Baan)". Alles wat we in de CRAB-straatassen terugvonden als Lievestraat is op dit moment herdoopt naar Liefkensweg. Graag horen we van u of we het voor de rest volledig juist hebben gedaan.
> 
> Het staat u volledig vrij deze kaart in te bouwen in uw website (door te linken of via de share-knop als iFrame). De data zelf kunt u ook hergebruiken (deze is eenvoudig downloadbaar), hoewel u verplicht bent om verbeterde versies van die data volgens dezelfde open data licentie te verspreiden als degene die we zelf gebruiken.
> 
> Met vriendelijke groet,
> Joost Schouppe
> Board member, OpenStreetMap Belgium
> www.osm.be
> community@osm.be

**Comment by pietervdvn** [comment]
Your mails are definitely more classy then my emails ;p


Met vriendelijke groeten,
Pieter Vander Vennet

2018-07-12 13:53 GMT+02:00 Joost <notifications@github.com>:

**Comment by pietervdvn** [comment]
Pelt confirmed our name changes to be correct.

**Comment by pietervdvn** [comment]
Aalter clarified a minor issue and invites us over for a meeting.

**Comment by joostschouppe** [comment]
Puurs is mapped.

**Comment by joostschouppe** [comment]
An idea for a "press release":
In januari 2019 gaat u schrijven over de moeilijkheden om de adreswijzigingen in de nieuwe fusiegemeenten door te geven aan de GPS- en kaarten-makers. In 2020 gaat u schrijven over hoe dat in een aantal toepassingen nog steeds niet in orde is. Maar u kunt ook nu al schrijven over hoe sommige kaartenmakers de nieuwe adressen al aan het gebruiken zijn, nog voor de officiële informatie helemaal rond is.

**Comment by xivk** [comment]
ha! :+1:

**Comment by pietervdvn** [comment]
Bump!

By now, some time has passed. Did we manage to finish everything?

Did we get feedback from every municipality? Did we rename all the houses if appropriate?
Are we still gonna do some press release and fancy map?

@joostschouppe: could you check off the relevant checkboxes and close the issue if everything has been done?

**Comment by joostschouppe** [comment]
I'd consider the project finished. I've checked off the boxes. Not all municipalities reacted, we could still send them a new e-mail. I think Midgard fixed up all the individual addresses.
At OpenBelgium I'll talk a bit about it; by then I'll try to do the blogpost part - after that we can close this Issue. I think we missed the opportunity for a press release by now, but we should keep a close look on the press if they talk about issues caused by the address changes.

**Comment by pietervdvn** [comment]
Alright! Then I'm gonna close it.
